### PR TITLE
WinDjView: Add version 2.1

### PR DESCRIPTION
**Pull request by taoari**
This pull request adds the WinDjView djvu reader to the bucket.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/windjview.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "2.1",
+    "description": "A fast, compact, and powerful DjVu viewer",
+    "homepage": "https://sourceforge.net/projects/windjview/",
+    "license": "GPL-2.0",
+    "url": "https://iweb.dl.sourceforge.net/project/windjview/WinDjView/2.1/WinDjView-2.1-Setup.exe#/dl.7z",
+    "hash": "sha1:742cee128eb976a372dcf9a1121eb0736e1df75d",
```
Comment (issaclin32) [inline bucket/windjview.json:7]:
```suggestion
    "hash": "sha1:742cee128eb976a372dcf9a1121eb0736e1df75d",
    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninstall.exe\" -Force -Recurse",
```

**Comment by rashil2000** [comment]
Use the `https://downloads.sourceforge.net/project/windjview` URLs?

File: bucket/windjview.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "bin": "WinDjView.exe",
```
[the remark this answers is not in the document]
Comment (issaclin32) [reply bucket/windjview.json:8]: I couldn't find too much information about it, but I found [this thread](https://sourceforge.net/p/windjview/discussion/392867/thread/d8314d72/) from 2007.
Therefore I assume the app **supports CLI args**.
```
In the next version the following syntax (similar to a link to a specific page) will be implemented:
WinDjView file.djvu#pageNo
```

[the remark this answers is not in the document]
Comment (rashil2000) [reply bucket/windjview.json:8]: Oh, interesting!

**Comment by taoari** [comment]
@issaclin32 @rashil2000 Thanks for the suggestion! The latest commit updates the pre_install and downloads urls.

**Review by rashil2000 — APPROVED** [review]

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**